exporter_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/name"
 	"github.com/sclevine/spec"
 	"github.com/sclevine/spec/report"
+	"github.com/stretchr/testify/assert"
```

Comment:
Can we avoid introducing testify? I think map marshaling + go-cmp can be used for json comparison.

Reply:
I'm sure we could. I do know that we've talked a bit with @ekcasey about introducing `testify` as a way to remove the overhead of needing to maintain our own sets of assertions, specially when having to do it in multiple projects. We thought this might be a good point in time to introduce it to `lifecycle`. It seems unnecessary to have to create basic assertion methods multiple times.

Reply:
After offline discussions this is being reverted and will might reconsider adding `testify` in the future.